plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialLevel.java

```diff
@@ -50,41 +51,59 @@ public static void registerTags() {
         // @returns ElementTag(Number)
         // @group properties
         // @description
-        // Returns the maximum level for a levelable material (like water, lava, and Cauldrons), a cake, or a beehive.
+        // Returns the maximum level for a level-able material (like water, lava, and cauldrons).

```

Comment:
Please either use `levelable` as was written, or `Levelled` to match the spigot API name.

Reply:
Also please restore the list of items back to the way it was.